### DISCUSSION TITLE
part-ii/2021-48/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#384] [Question comments]
Part II 2021 Differential Geometry: Paper 2, Section II, $26 \mathbf{F}$ 

https://questions.tripos.org/part-ii/2021-48/

**Comment by ArthurConmy** [comment]
I get 

<details>
<summary>Spoiler</summary>

I get $f(u) = Ae^{u} + Be^{-u}, g(u) = 2 \sqrt{AB} u + C$ where $A,B$ have the same sign (and $A,B,C \in \mathbb{ R}$).

</details>
